docs/book/OWNERS

```diff
@@ -1,5 +1,5 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 reviewers:
-  - randomvariable
-  - moshloop
+- cluster-api-reviewers
+- cluster-api-book-reviewers
```

Comment:
Since we are not setting options.no_parent_owners: true, you can exclude anyone listed as an approver/reviewer for the top level OWNERS file, since it will be inherited by default. https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md#owners